content/en/docs/concepts/workloads/pods/ephemeral-containers.md

```diff
@@ -76,3 +76,267 @@ you can view processes in other containers.
+
+One notable feature of `ephemeralContainers` is its ability to configure the container security context. This means that in theory, a malicious actor could potentially escalate the container's privileges. However, the impact of this depends on the policy enforcement tool being used. If you are using a robust policy enforcement tool, it should be able to mitigate such risks.
+
+The answer to whether it affects users of policy enforcement tools varies. It depends on the specific tool and its version, as well as whether you are using policies from the project's library or custom policies developed in-house. To ensure security, it is advisable to review the policies and keep the policy enforcement tool up-to-date with the latest security measures.
+
+
+## OPA Gatekeeper
+
+OPA Gatekeeper does not require any code changes as all of its policies are 
+written in rego. It's sub project Gateekper Library does however have to be 
+updated. The library contains an implementation of the common Pod Security
+Policies. This includes policies like not allowing containers in privileged 
+mode. The issue with the all of the policies is that they currently only 
+check containers specified in initContainers and containers, analyze the 
+following rego as an example.
+
+The good news is that this is a pretty easy fix, the bad news is
+that it requires end users to update the policies pulled from the library.
+
+
+## Pod Security Policies
+
+Pod Security Policies used to be the default policy tool for Kubernetes,
+and a lot of projects have rules based on Pod Security Policies (PSP).
+However if you are relying on PSP in a modern cluster you should really
+start looking for other options like OPA Gatekeeper or Kyverno. 
+PSP has been deprecated since Kubernetes v1.21 and will be removed in v1.25.
+
+If PSP is your only policy tool and you are planning to upgrade to v1.23,
+don't. As PSP is deprecated no new features have been added, and that
+includes policy enforcement on ephemeral containers. Which means that
+any security context in an ephemeral container is allowed no matter 
+the PSP in the cluster. The PSP below will have no affect when adding
+an ephemeral container to a Pod which is privileged.  
+   
+```yaml
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: default
+spec:
+  privileged: false
+  seLinux:
+    rule: RunAsAny
+  supplementalGroups:
+    rule: RunAsAny
+  runAsUser:
+    rule: RunAsAny
+  fsGroup:
+    rule: RunAsAny
+  volumes:
+  - '*'
+
+```
+
+## RBAC
+
+Disallowing ephemeral containers with RBAC could be an option if
+the feature is not needed and it is not possible to disable the
+feature completely. The KEP-277: Ephemeral Containers state the
+following about using RBAC to disable ephemeral containers.
+
+Cluster administrators will be expected to choose from one of the
+following mechanisms for restricting usage of ephemeral containers:
```

Comment:
Is it possible to format this as a quote in markdown?

Either way it would be good to link to the source, https://git.k8s.io/enhancements/keps/sig-node/277-ephemeral-containers#configurable-security-policy

Reply:
Even better: rewrite the quoted text to match our docs [style guide](https://kubernetes.io/docs/contribute/style/style-guide/).

“As a cluster operator, you are expected to choose from one of” for example.